srcpkgs/Signal-Desktop/template

```diff
@@ -1,22 +1,28 @@
 # Template file for 'Signal-Desktop'
 pkgname=Signal-Desktop
-version=1.39.5
+version=5.0.0
 revision=1
 # Due to electron
 # 32-bit is not supported https://github.com/signalapp/Signal-Desktop/issues/1661
 archs="x86_64"
-hostmakedepends="git nodejs python tar yarn"
+hostmakedepends="git git-lfs nodejs python tar yarn"
 depends="libnotify libappindicator"
 short_desc="Signal Private Messenger for Linux"
 maintainer="Julio Galvan <juliogalvan@protonmail.com>"
 license="GPL-3.0-only"
 homepage="https://github.com/signalapp/Signal-Desktop"
 distfiles="https://github.com/signalapp/Signal-Desktop/archive/v${version}.tar.gz"
-checksum=f9f55c0195579b43712bf370b753337432eda4bcf89809d1c8f3deb0b4ee4400
+checksum=4ff58f5ab5f4de6477dd92069d8c2e9a830513acca527b51e0ee1c7cbe7d8627
 nostrip_files="signal-desktop"
 
 pre_build() {
+	git lfs install
```

Comment:
I don't know if this is actually ok to do here. What files does it touch?

Reply:
@ericonr I don't know, but it's what I needed to get any version past what's in the repos to build 🤷

Reply:
Man page says

```
DESCRIPTION
Perform the following actions to ensure that Git LFS is setup properly:

○
    Set up the clean and smudge filters under the name "lfs" in the global Git config.
○
    Install a pre-push hook to run git-lfs-pre-push(1) for the current repository, if run from inside one. If "core.hooksPath" is configured in any Git configuration (and supported, i.e., the installed Git version is at least 2.9.0), then the pre-push hook will be installed to that directory instead.
```

@void-linux/pkg-committers is this ok or should we change the position of the git config directory or something?

Reply:
Anyway, IMO this specific command should be done in `pre_configure`

Reply:
if it touches anything outside builddir it's not okay